firestore-bigquery-export/extension.yaml

```diff
@@ -206,6 +206,19 @@ params:
     default: posts
     required: true
 
+  - param: LOG_FAILED_EXPORTS
+    label: Enable logging failed exports
+    description: >-
+      If enabled, the extension will exports that failed to enqueue to the
+      Firebase console, to prevent data loss.
```

Comment:
probably "to mitigate" is better than "prevent"? the logs won't prevent it entirely, just helps see what failed.

Reply:
yup, will change! good catch